blazor/pdfviewer/getting-started/wsl-application.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: Getting Started with PDF Viewer in Blazor WSL mode | Syncfusion
+description: Learn how to getting started with PDF Viewer control in Blazor WSL (Windows Subsystem for Linux) mode. 
+platform: Blazor
+control: PDF Viewer
+documentation: ug
+---
+
+# Getting Started with Blazor PDF Viewer Component in WSL mode
+
+To run the Syncfusion Blazor PDF Viewer in WSL (Windows Subsystem for Linux) mode, you will need to take the following steps:

```

Comment:
To run the Syncfusion Blazor PDF Viewer in WSL (Windows Subsystem for Linux) mode, follow these steps: